Include/object.h

```diff
@@ -239,6 +239,9 @@ PyAPI_FUNC(PyObject*) PyType_FromModuleAndSpec(PyObject *, PyType_Spec *, PyObje
 PyAPI_FUNC(PyObject *) PyType_GetModule(struct _typeobject *);
 PyAPI_FUNC(void *) PyType_GetModuleState(struct _typeobject *);
 #endif
+#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x03110000
```

Comment:
3.11 is `030B` in hex, not `0311`. 
```suggestion
#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x030B0000
```

Reply:
ops. Thanks.